pyodide_build/pywasmcross.py

```diff
@@ -608,7 +616,7 @@ def handle_command(
 
         scipy_fixes(new_args)
 
-    result = subprocess.run(new_args)
+    result = subprocess.run(new_args, check=True)
```

Comment:
We check the result ourselves
```suggestion
        check=False
```